test/language/statements/async-function/let-newline-await-in-async-function.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2017 Mozilla Corporation. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Jeff Walden <jwalden+code@mit.edu>
+esid: sec-let-and-const-declarations
+description: >
+  |await| is excluded from LexicalDeclaration by grammar parameter, in
+  AsyncFunction.  Therefore |let| followed by |await| inside AsyncFunction is
+  an ASI opportunity, and this code must parse without error.
+---*/
+
+async function f() {
+    let
```

Comment:
Sure it is -- `ExpressionStatement` lookahead forbids `let [`, but it doesn't forbid `let` followed by some token other than `[`.  And plain old `let` is a perfectly cromulent `PrimaryExpression`.